### ISSUE TITLE
Pass root answers.conf to inherited applications.

### ISSUE THREAD

**Issue by cdrage** [projectatomic/atomicapp#447]
As per @vpavlin 's comment on https://github.com/projectatomic/nulecule/issues/185

We should be passing the root answers.conf to all inherited applications

Specifically, this line of code:

https://github.com/projectatomic/atomicapp/blob/master/atomicapp/nulecule/base.py#L313

Thus when deploying an inherited application, instead of using the default namespace (currently just `default`) we will use the namespace that has been provided within `answers.conf`.

**Comment by rtnpro** [comment]
This will be taken care by #423

**Comment by rtnpro** [comment]
@cdrage 

Currently, we do pass the entire config data to all the nodes. So, a node can always checkup for the ``namespace`` param in ``config['general']``.

**Comment by rtnpro** [comment]
@cdrage ping?

**Comment by cdrage** [comment]
@rtnpro Yeah, you're right, but we don't pass _all_ the information yet (for ex. if we add params about persistent storage and such onto the graph it's not passed onto the provider) just the general configuration information.

**Comment by rtnpro** [comment]
@cdrage because persistent storage is not part of the config data, right now :) We need to syncup to discuss on this.
